test/k8s-integration/driver-config.go

```diff
@@ -72,9 +71,18 @@ func generateDriverConfigFile(pkgDir, storageClassFile, deploymentStrat string)
 		return "", fmt.Errorf("got unknown deployment strat %s, expected gce or gke", deploymentStrat)
 	}
 
+	var absSnapshotClassFilePath string
+	// If snapshot class is passed in as argument, include snapshot specific driver capabiltiites.
+	if snapshotClassFile != "" {
+		caps = append(caps, "snapshotDataSource", "pvcDataSource")
```

Comment:
pvcDataSource refers to the "cloning" feature, which pd driver does not support.

Reply:
Ack. Will fix and retest.

Reply:
Enabling the test should have caused a test failure since the driver doesn't support it. I'm curious to understand why it passed when you were testing it?

Reply:
I was testing only focussed snapshot tests using "External.Storage.*snapshot" instead of "External.Storage", hence only the snapshot tests were run. 
I will now run with just "External.Storage" and see the test results.